csrankings-k.csv

```diff
@@ -833,7 +833,7 @@ Kun Sun 0001,George Mason University,https://csis.gmu.edu/ksun,u18hG4gAAAAJ
 Kun Xie 0001,Hunan University,https://cskxie.github.io,2cfH6_0AAAAJ
 Kun Xu 0003,Tsinghua University,http://cg.cs.tsinghua.edu.cn/people/~kun,dRlBzscAAAAJ
 Kun Yang 0001,University of Essex,https://www.essex.ac.uk/people/yangk82907/kun-yang,qpe3iUkAAAAJ
-Kun Zhang 0001,Carnegie Mellon University,https://www.cmu.edu/dietrich/philosophy/people/faculty/zhang.html,RGoypN4AAAAJ
+Kun Zhang 0001,MBZUAI,https://mbzuai.ac.ae/study/faculty/dr-kun-zhang,RGoypN4AAAAJ

```

Comment:
Website says "Kun Zhang has joined MBZUAI as Associate Professor in the Machine Learning Department. He also maintains an associate professorship at Carnegie Mellon University." Please verify that this is in fact a full-time appointment at MBZUAI (CMU page doesn't indicate change to MBZUAI affiliation).